### DISCUSSION TITLE
Bundling static localization json files for i18next

### DISCUSSION THREAD

**Discussion by bmmpt** [parcel-bundler/parcel#6240] [Q&A] [answered]
I use i18next in my react application for localization. I have a folder with a series of json resource files that I need bundled. These files are not imported in the code. Only when needed i18next uses fetch to to get the files, and only the exact file it needs, not all of them.

With parcel 1 I used the plugin parcel-plugin-static-files-copy with the following config:

"staticFiles": { "staticPath": "public/locales" }

That would copy/bundle my translation/resource files to the /dist folder, so when i18next tried to "fetch" the urls resulting from

loadPath: `${location.protocol}//${location.host}/{{lng}}/{{ns}}.{{lng}}.json`, 

the files would be present/served by the server.

Since I don't actually "import" my translation/resource .json files in my js code, they don't get bundled by Parcel 2... Does Parcel 2 have a way of doing what parcel-plugin-static-files-copy did for Parcel 1?

**Comment by mischnic** [comment, marked as answer]
The readme of that Parcel 1 plugin mention a Parcel 2 version: https://github.com/elwin013/parcel-plugin-static-files-copy#readme
